examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -14,6 +14,13 @@ import { DiceRollerController } from "./controller";
 import { ConsoleLogger } from "./ConsoleLogger";
 import { renderAudience, renderDiceRoller } from "./view";
 
+export interface ICustomUserDetails {
+    gender?: string;
+    email?: string;
+}
+
+const userDetails: ICustomUserDetails = {};
```

Comment:
Lets fill the userDetails in with some sample values and then we can also drop the optional ? in the fields in ICustomUserDetails. Otherwise, the example doesn't actually show the userDetails being used